.github/workflows/docs.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   test:
-    uses: FuelLabs/github-actions/.github/workflows/mdbook-docs.yml@master
+    uses: FuelLabs/github-actions/.github/workflows/mdbook-docs.yml@mdbook-docs-pre-command
```

Comment:
After https://github.com/FuelLabs/github-actions/pull/21 is merged, this will change back to `master`.